### PR TITLE
[FIX][DOC] fix doc build

### PR DESCRIPTION
**Pull request by Remi-Gau**
- test doc build in CI so that it tested on PR and not just on master
- 'silence' sphinx gallery that seems to be causing the doc build issues
- use same build tool orchestration (make) locally and in CI

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -3,9 +3,9 @@
 
 # You can set these variables from the command line, and also
 # from the environment for the first two.
-SPHINXOPTS    ?=
+SPHINXOPTS    ?= --verbose --nitpicky
```
Comment (Remi-Gau) [inline docs/Makefile:6]:
```suggestion
SPHINXOPTS    ?= --verbose
```

**Comment by Remi-Gau** [comment]
The doc can now build but I had to 'nuke' sphinx gallery to make it work. Could not yet identify the root cause of this.

Given the example in question I would suggest, keeping things that way for now so there is functioning doc build and fix the gallery later.

**Comment by paquiteau** [comment]
Hello Remi ! Thanks for the PR  :) 

Yeah the gallery can stay in oblivion until we figure out a proper example setup

**Review by paquiteau — APPROVED** [review]

**Comment by Remi-Gau** [comment]
OK the doc builds but it is not deployed... will try to fix on my fork

**Pull request merged**